tests/test_asgi.py

```diff
@@ -29,7 +29,7 @@ async def events() -> AsyncIterator[dict]:
 
 @pytest_asyncio.fixture
 async def client(app: ASGIApp) -> AsyncIterator[httpx.AsyncClient]:
```

Comment:
```suggestion
async def client(app: Any) -> AsyncIterator[httpx.AsyncClient]:
```

Reply:
I'd like to mark it as `type: ignore[arg-type]` because `httpx.ASGITransport` uses its own ASGI typing, which differs from `starlette`'s.

Reply:
I understand both ways will work!

You should be able run `make format` locally to fix the Black formatting error

Reply:
Yep thanks for that!